<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="58" started-at="2017-03-25T04:56:01Z" finished-at="2017-03-25T04:56:01Z">
    <groups>
      <group name="High">
        <method signature="test_case1.testcase1()[pri:0, instance:testing.test_case1@629f0666]" name="testcase1" class="testing.test_case1"/>
      </group> <!-- High -->
      <group name="Low">
        <method signature="test_case1.testcase2()[pri:0, instance:testing.test_case1@629f0666]" name="testcase2" class="testing.test_case1"/>
      </group> <!-- Low -->
    </groups>
    <test name="Default test" duration-ms="58" started-at="2017-03-25T04:56:01Z" finished-at="2017-03-25T04:56:01Z">
      <class name="testing.test_case1">
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:testing.test_case1@629f0666]" name="beforemethod" is-config="true" duration-ms="12" started-at="2017-03-25T10:26:01Z" finished-at="2017-03-25T10:26:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="PASS" signature="testcase1()[pri:0, instance:testing.test_case1@629f0666]" name="testcase1" duration-ms="7" started-at="2017-03-25T10:26:01Z" finished-at="2017-03-25T10:26:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase1 -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:testing.test_case1@629f0666]" name="aftermethod" is-config="true" duration-ms="0" started-at="2017-03-25T10:26:01Z" finished-at="2017-03-25T10:26:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:testing.test_case1@629f0666]" name="beforemethod" is-config="true" duration-ms="1" started-at="2017-03-25T10:26:01Z" finished-at="2017-03-25T10:26:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="FAIL" signature="testcase2()[pri:0, instance:testing.test_case1@629f0666]" name="testcase2" duration-ms="1" started-at="2017-03-25T10:26:01Z" finished-at="2017-03-25T10:26:01Z">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problem: 
	The method assertTrue(boolean) in the type Assert is not applicable for the arguments (String)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problem: 
	The method assertTrue(boolean) in the type Assert is not applicable for the arguments (String)

	at testing.test_case1.testcase2(test_case1.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase2 -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:testing.test_case1@629f0666]" name="aftermethod" is-config="true" duration-ms="0" started-at="2017-03-25T10:26:01Z" finished-at="2017-03-25T10:26:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
      </class> <!-- testing.test_case1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
